.github/workflows/scripts/dist-diff.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Make the whole script fail if any of the commands fail
+# (including within pipes, in case we add some in the future)
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+
+# Computes the diff of `dist` between a provided base reference (branch, tag, commit SHA) and the current HEAD.
+# It outputs the resulting diff in the provided output folder
+# 
+# Usage: `dist-diff.sh <base-reference> <output-folder>`
+
+# Default the base branch to main to allow running locally quickly
+base="${1:-main}"
+# And the output folder to the current working directory
+output_folder="${2:-`pwd`}"
+
+# Diff the minified JS file
+git diff \
+  $base:dist/govuk-frontend-$(git show $base:dist/VERSION.txt).min.js \
+  dist/govuk-frontend-$(cat dist/VERSION.txt).min.js \
+  > $output_folder/dist-js.diff
+# Diff the minified CSS file
```

Comment:
Mind if we keep the line breaks above comments (as above?)

Makes it a bit more readable

```suggestion

# Diff the minified CSS file
```